### PR TITLE
Feature/plcn 381 create get current user query

### PR DESCRIPTION
**Pull request by AntonEsbjergIT-Minds**
Created Get current user query.

### CODE REVIEW

File: test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+﻿using AutoMapper;
```
Comment (itm-mne) [inline test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs:1]: Arent these files usually named something with `test`?

**Review by itm-mne — APPROVED** [review]
Consider renaming one of the files according to the comment (Only if my assumption is corrent)

File: test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+using Pelican.Application.Abstractions.Authentication;
+using Pelican.Application.Abstractions.Data.DataLoaders;
+using Pelican.Application.Authentication;
+using Pelican.Application.Users.Queries.GetCurrentUser;
+using Pelican.Domain.Entities;
+using Pelican.Domain.Entities.Users;
+using Xunit;
+
+namespace Pelican.Application.Test.Users.Queries;
+public class GetCurrentUserQueryHandlerUnitTest
```
Comment (Nicolai-Bak) [inline test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs:12]: Match filename with classname

File: test/Pelican.Presentation.GraphQL.Test/UsersQueryUnitTest.cs [hunk 3]
```diff
@@ -21,4 +22,16 @@ public void GetUsersAsync_MediatorRecievesSendCallWithAGetUsersQueryasParam_NoEr
 			x => x.Send(It.IsAny<GetUsersQuery>(), default),
 			Times.Once);
 	}
+
+	[Fact]
+	public void GetCurrentUserAsync_MediatorRecievesSendCallWithAGetCurrentUserQueryAsParam_NoErrors()
```
Comment (Nicolai-Bak) [inline test/Pelican.Presentation.GraphQL.Test/UsersQueryUnitTest.cs:27]: Receives *

File: test/Pelican.Presentation.GraphQL.Test/UsersQueryUnitTest.cs [hunk 4]
```diff
@@ -11,7 +12,7 @@ public class UsersQueryUnitTest
 	private readonly Mock<IMediator> _mediatorMock = new();
 
 	[Fact]
-	public void GetUsersAsync_MediatorRecievesSendCallWithAGetUsersQueryasParam_NoErrors()
+	public void GetUsersAsync_MediatorRecievesSendCallWithAGetUsersQueryAsParam_NoErrors()
```
Comment (Nicolai-Bak) [inline test/Pelican.Presentation.GraphQL.Test/UsersQueryUnitTest.cs:15]: Receives *

File: test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs (hunk 1 above)
Comment (AntonEsbjergIT-Minds) [reply test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs:1]: Yes, It has been renamed

File: test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs (hunk 2 above)
Comment (AntonEsbjergIT-Minds) [reply test/Pelican.Application.Test/Users/Queries/GetCurrentUserQuery.cs:12]: Renamed

**Review by Nicolai-Bak — APPROVED** [review]

**Pull request merged**